spring-boot-dependencies/pom.xml

```diff
@@ -2285,6 +2286,26 @@
 				<version>${jooq.version}</version>
 			</dependency>
 			<dependency>
+				<groupId>org.jetbrains.kotlin</groupId>
+				<artifactId>kotlin-reflect</artifactId>
+				<version>${kotlin.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.jetbrains.kotlin</groupId>
+				<artifactId>kotlin-stdlib</artifactId>
```

Comment:
Doesn't `kotlin-stdlib-jre8` transitively pull `kotlin-stdlib` and `kotlin-stdlib-jre7`?